conda-forge.yml

```diff
@@ -1,5 +1,8 @@
 bot:
   automerge: true
+  abi_migration_branches:
+  - "8.3xx"
+  - "8.311"
```

Comment:
I added https://github.com/conda-forge/pythia8-feedstock/tree/8.311 to be able to do rebuilds for `mg5amcnlo` `v3.5.7` (c.f. https://github.com/conda-forge/mg5amcnlo-feedstock/issues/2), so this should also get ABI migrations.

However, this isn't building for some reason, which is confusing as the [conda-forge maintainers docs say](https://conda-forge.org/docs/maintainer/updating_pkgs/#maintaining-several-versions)

> Then push this branch from your fork to the upstream feedstock. Our CI services will automatically build any branches in addition to the default branch.

I did this and it isn't building. :?

Reply:
```
git commit --amend --no-edit && git push --force-with-lease
```

has it building. Seems I got unlucky with my original push timing and the event got missed.